docs/pbc/all/search/202410.0/base-shop/third-party-integrations/algolia/algolia-personalization-for-headless-frontends.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Algolia Personalization for headless frontends
+description: Find out how you can enable Algolia personalization in your Spryker shop based on headless approach (custom frontend or mobile application). 
+last_updated: Nov 24, 2024
+template: howto-guide-template
+---
+
+{% info_block infoBox "Info" %}
+
+Default Spryker installation supports Algolia personalization only for YVES frontend.
+
+{% endinfo_block %}
+
+The events from your site/application are required to enable Algolia premium features such as
```

Comment:
Do you mean that the events need to be sent to Algolia?

Reply:
right